ProjFS.Mac/Scripts/Build.sh

```diff
@@ -81,6 +89,38 @@ while read line; do
 	 [[ $line != *"RetainIOCount"* ]] && 
 	 [[ $line != *"ProviderMessaging_"* ]] && 
 	 [[ $line != *"RWLock_DropExclusiveToShared"* ]] && 
+	 [[ $line != *"PrjFS_"* ]] &&                                     #SHOULD ADD COVERAGE
```

Comment:
There are a lot of exclusions now.  Can we break it up a bit with a comment
##### PrjjFsLib Excusions ######
Same for PrjFsKext